js/script.spec.js

```diff
@@ -48,5 +48,13 @@ describe("Beer display module", () => {
         expect(beerContainer.innerHTML.toString().includes(beer.name)).toBeTruthy();
         expect(beerContainer.innerHTML.toString().includes('span')).toBeTruthy();
     });
+
+    test("createImgContainer changes DOM, display has class `imgContainer` and `image_url` is visible in element", () => {
```

Comment:
are you sure this is correct name?

Reply:
Maybe it should be name like this 

`createImgContainer changes DOM, element with class `imgContainer` is visible and `image_url` value is inside element`